### DISCUSSION TITLE
función confusa

### DISCUSSION THREAD

**Discussion by frankystanley** [community/community#63773] [Discussions]
### Select Topic Area

General

### Body

Tengo que escribir una función llamada convertirANumerosRomanos que tome un número decimal (entre 1 y 3999) como argumento y lo convierta en su representación en números romanos. Utiliza los símbolos romanos básicos (I, V, X, L, C, D, M) y sigue las reglas de formación de números romanos y estoy perdido

**Comment by EnriquezAlberto** [comment]
```
function convertirANumerosRomanos(numero) {
  const valores = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
  const simbolos = ["M", "CM", "D", "CD", "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"];

  let resultado = "";
  for (let i = 0; i < valores.length; i++) {
    while (numero >= valores[i]) {
      resultado += simbolos[i];
      numero -= valores[i];
    }
  }
  return resultado;
}
```

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
